newdle/config.py

```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint, current_app
+
+
+config = Blueprint('config', __name__, url_prefix='/config')
+
+
+@config.route('/footer_links/')
+def footer_links():
+    if 'FOOTER_LINKS' in current_app.config:
```

Comment:
BTW, in case this check was necessary, `current_app.config.get('FOOTER_LINKS', {})` would be a more pythonic alternative to these 4 lines here